etc/databases/world/updates/updates.sql

```diff
@@ -998,5 +998,176 @@ begin not atomic
+        -- Add Unholy Avenger to Dreadlord Malganis, who seems to be the only reasonable source for this item.
+        DELETE FROM `creature_loot_template` WHERE (`entry` = 929) AND (`item` IN (3687));
+        INSERT INTO `creature_loot_template` (`entry`, `item`, `ChanceOrQuestChance`, `groupid`, `mincountOrRef`, `maxcount`, `condition_id`) VALUES (929, 3687, 20, 0, 1, 1, 0);
+
+        insert into applied_updates values ('111120241');
+    end if;
+
+    -- 22/11/2024 1
+    if (select count(*) from applied_updates where id='111120241') = 0 then
```

Comment:
Seems like you are using the wrong id here (should be `221120241`) and also forgot to add `insert into applied_updates values ('221120241');` at the end of the update block.